repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publications-overview.ftl

```diff
@@ -13,11 +13,11 @@
 
 
 <#-- CONTENT: -->
-<section class="document-content">
+<section class="document-content" aria-label="Publication Overview">
```

Comment:
I'm definitely not the right person to lecture native speakers on English, but wouldn't this be more accurate if it said 'Publication**s** Overview'?

Reply:
Good spot! Will update 🙂

Reply:
Cool! Feel free to merge straight after, I don't need to see the fix :wink: